src/api/product-mock-ec/api-mock-ec/v1/_base_policy.xml

```diff
@@ -0,0 +1,27 @@
+<!--
+    IMPORTANT:
+    - Policy elements can appear only within the <inbound>, <outbound>, <backend> section elements.
+    - To apply a policy to the incoming request (before it is forwarded to the backend service), place a corresponding policy element within the <inbound> section element.
+    - To apply a policy to the outgoing response (before it is sent back to the caller), place a corresponding policy element within the <outbound> section element.
+    - To add a policy, place the cursor at the desired insertion point and select a policy from the sidebar.
+    - To remove a policy, delete the corresponding policy statement from the policy document.
+    - Position the <base> element within a section element to inherit all policies from the corresponding section element in the enclosing scope.
+    - Remove the <base> element to prevent inheriting policies from the corresponding section element in the enclosing scope.
+    - Policies are applied in the order of their appearance, from the top down.
+    - Comments within policy elements are not supported and may disappear. Place your comments between policy elements or at a higher level scope.
+-->
+<policies>
+    <inbound>
+        <base />
+        <rewrite-uri template="/" />
```

Comment:
@pasqualedevita without modifying the app through the policy I can remove the path in request. So it goes 🚀 

```sh
curl --location --request POST 'https://api.dev.platform.pagopa.it/mock-ec/api/paVerifyPaymentNotice' \
--header 'Content-Type: application/xml' \
--data-raw '<soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/" xmlns:nod="http://pagopa-api.pagopa.gov.it/node/nodeForPsp.xsd">
  <soapenv:Header />
  <soapenv:Body>
    <pafn:paVerifyPaymentNoticeReq>
      <idPA>77777777777</idPA>
      <idBrokerPA>77777777777</idBrokerPA>
      <idStation>77777777777_01</idStation>
      <qrCode>
        <fiscalCode>77777777777</fiscalCode>
        <noticeNumber>302989852145637231</noticeNumber>
      </qrCode>
    </pafn:paVerifyPaymentNoticeReq>
  </soapenv:Body>
</soapenv:Envelope>'
```sh

Reply:
Top! Before merge we can try to improve api and product dir